### DISCUSSION TITLE
What about merging reports?

### DISCUSSION THREAD

**Discussion by ice1000** [lefou/mill-jacoco#35] [Q&A] [answered]
Can we merge several reports into one? Similar to https://github.com/gradle/gradle/issues/10708#issuecomment-566279825, you simply put the execution data together.

**Comment by ice1000** [comment]
I mean I dunno how to do it with this plugin

**Comment by lefou** [comment, marked as answer]
Sorry, for not answering earlier. I need to look into JaCoCo again, to see how to do it, but am currently a bit limited in time. Maybe it already works but is not documented. You could also look at the sources and play around yourself. (https://github.com/lefou/mill-jacoco/blob/main/core/src/de/tobiasroeser/mill/jacoco/JacocoReportModule.scala#L96) Maybe, you just need to add more jacoco.exec files.

**Comment by ice1000** [comment]
Don't worry, I'm asking a question, not asking you to do it for me. It's good to have your answer!
